views.py

```diff
@@ -15,38 +15,38 @@
 
 def split_races(results):
 
-	races = []
+    races = []
 
-	for x in range(1, 8):
+    for x in range(1, 8):
 
-		race = []
+        race = []
 
-		for result in results:
-			if result[2] is x:
-				race.append(result)
+        for result in results:
+            if result[2] is x:
+                race.append(result)
 
-		races.append(race)
+        races.append(race)
 
-	return races
+    return races
```

Comment:
Sigh, the one disagreement I have with pep8.

I can get away with doing that on private projects, but I guess we'd better stick to the style guide here.